examples/pool.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.push File.expand_path("../lib", __dir__)
+require "celluloid/autostart"
+
+# This is required for pool to work as Celluloid doesn't load it by default
+require "celluloid/pool"
+require "prime"
```

Comment:
shall we have empty line here?

```
# This is required for pool to work as Celluloid doesn't load it by default
require "celluloid/pool"

require "prime"
```
Just a very small improvement, not doing it also fine

Reply:
Sure thing! Will push it soon